docs/data_structures.md

```diff
@@ -0,0 +1,74 @@
+Taking the latitude of each task location into account the satellite imagery has a maximum spatial resolution of ~ 0.6 meter at the equator.
+
+### For building footprint validation projects
+
+A task is the validation of the footprint of a single building. This would be a single "way" in OSM terminology. With the data used in input, multipolygons shouldn't appear, and if they do, they will be ignored here. Similarly, a single node will be ignored.
+The data required to complete a task includes:
+- the building geometry: 
+
+| Parameter | Description | Example: Built areas | Example: Building footprint |
+| --- | --- | --- | --- |
+| **Id** | Each task can be identified by its Id. The Id is a composition of its position in the corresponding tile map system, which can be described by the x, y and z coordinates. | "18-160175-127032" |  |
+| **Project Id** | The Id of the project this task relates to | "1" |  |
+| **Tile Z** | The z coordinate of the tile defines the zoom level. Greater values for z will correspond to higher spatial resolution of the corresponding image. For most regions Bing provides images up to zoom level 18. For aerial imagery or images captured by UAVs even higher z values are valid. | "18" | Similar to existing projects: These can be determined by the worker scripts, so the app has little to do here. |
+| **Tile X**| The x coordinate characterises the longitudinal position of the tile in the overall tile map system taken the zoom level into account. The x coordinates increase from west to east starting at a longitude of -180 degrees. | "160175" | See above |
+| **Tile Y** | The y coordinate characterises the latitudinal position of the tile in the overall tile map system taken the zoom level into account. The latitude is clipped to range from circa -85 to 85 degrees. The y coordinates increase from north to south starting at a latitude of around 85 degrees. | "127032" | See above |
+| **Footprint** | The list of nodes constituting the footprint corners. There must be at least 3 nodes. | Not used | [(x1, y1), (x2, y2), (x3, y3), (x4, y4)] |
+| **Geometry** | Each task has a polygon geometry, which can be generated by its x, y and z coordinates. At the equator the task geometry is a square with an edge length of around 150 metres covering circa 0.0225 square kilometres. Due to the web Mercator projector the task geometry will be clinched with increasing distance to the equator. At the same time the area per task will decrease. | Unused? | |
+| **Tile URL** | The tile URL points to the specific tile image described by the x, y, and z coordinates. Usually, the image has a resolution of 256 x 256 pixels. However, some providers also generate image tiles with higher resolution (e.g. 512 x 512 pixels). | "http://t0.tiles.virtualearth.net/tiles/a122333000110323111.jpeg?g=854&mkt=en-US&token=AopsdXjtTu-IwNoCTiZBtgRJ1g7yPkzAi65nXplc-eLJwZHYlAIf2yuSY_Kjg3Wn" | idem? |
```

Comment:
instead of adding the final link to the tile, we could add just the name of the tileserver that's going to be used. This would also help us to hide the api key somewhere inside the app. 

This will also be related to how the satellite imagery is displayed in the app. e.g. using leaflet or just displaying images.